controllers/user.controller.js

```diff
@@ -225,7 +226,7 @@ export const createAdmin = async (req, res) => {
 };
```

Comment:
`res.render` is invalid as we are not using EJS yet, but either way the API should only ever send a JSON response

Reply:
True

Reply:
this is a old method to create admin . I am not going to use that , will create another issue and delete unwanted methods and code